src/CodeCompliance/Domain/Checks/PrivateApi/Used/DependencyInBusinessModel.php

```diff
@@ -36,6 +37,14 @@ public function getGuideline(): string
         return 'Please avoid usage of %s in %s';
     }
 
+    /**
+     * @return string
+     */
+    public function getDocumentationUrl(): string
+    {
+        return static::DOCUMENTATION_BASE_URL . 'private-api-is-extended.html';
```

Comment:
might be more appropriate
https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-extended.html#example-of-code-that-causes-an-upgradability-error-extending-a-private-api-business-model

Reply:
\+